### PR TITLE
Add support to OMG's OCL language

### PR DESCRIPTION
**Pull request by sunye**
Dear rouge team,

I just implemented a Lexer for the last specificaton (2.4) of the OMG's Object Constraint Language. 

Thank you for this great piece of software.

Regards,

Gerson

### CODE REVIEW

File: lib/rouge/lexers/ocl.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          union intersection including excluding symmetricDifferencecount
+          append prepend insertAt subOrderedSet first last reverse subSequence
+          any closure collect collectNested exists forAll isUnique iterate one
+          reject select sortedBy allInstances average conformsTo
+        )
+      end
+
+      state :single_string do
+        rule %r/\\./, Str::Escape
+        rule %r/''/, Str::Escape
```
Comment (pyrmont) [inline lib/rouge/lexers/ocl.rb:55]: So to confirm, if you write `'this''is a string'` that gets treated as a single string rather than two strings?

**Review by pyrmont — CHANGES_REQUESTED** [review]
Thanks for the submission. Just had one question:

File: lib/rouge/lexers/ocl.rb (hunk 1 above)
Comment (sunye) [reply lib/rouge/lexers/ocl.rb:55]: No, this is wrong, it should be treated as two strings

Comment (pyrmont) [reply lib/rouge/lexers/ocl.rb]: I've removed this rule. That's all right then?

Comment (sunye) [reply lib/rouge/lexers/ocl.rb]: Yes, thank you!

**Review by pyrmont — APPROVED** [review]

**Pull request merged**